README.md

```diff
@@ -60,6 +57,14 @@ Devtron is a Web-Based CI/CD Platform for Kubernetes. It integrates various Open
 
 <p align="center"><img src="./assets/readme-comic.png"></p>
 
+<b> Devtron is built in a modular approach. These modules can be installed independently: </b>
+- [Full Devtron Experience](https://github.com/devtron-labs/devtron#tada-Full-Devtron-Experience) - This option gives you all the features of Devtron as a Full Experience providing you with CI, CD, security, cost, observability, stabilization. All the modules stated below are included here.
```

Comment:
is it ok to call it full devtron experience? it seems like other has some capabilities missing... shouldnt we position as two products? may be we can have section on when to use which